frontend/src/Judge/components/Submission.js

```diff
@@ -11,26 +11,30 @@ const PHOTO = 'PHOTO'
 const OTHER = 'OTHER'
 
 const Submission = props => (
-  <div>
+  <div style={{ height: '100%' }}>
```

Comment:
So, there's some stuff that may help with the spacing, in this file I did this: 
![image](https://user-images.githubusercontent.com/8568993/37809363-5e7c3210-2e25-11e8-8280-631cf6a67137.png) which gives the submission some breathing room like this: 
![image](https://user-images.githubusercontent.com/8568993/37809391-7e5df5b4-2e25-11e8-9eec-f38bcb5fdfe6.png)

Reply:
This didn't end up working for me. I learned how to use flexbox and fixed it that way.